tools/run_tests/run_tests.py

```diff
@@ -56,7 +56,7 @@
 )
 sys.path.append(gcp_utils_dir)
 
-_ROOT = os.path.abspath(os.path.join(os.path.dirname(sys.argv[0]), "../.."))
+_ROOT = os.path.abspath(os.path.join(os.path.dirname(sys.argv[0]), "..", ".."))
```

Comment:
Do we need the changes in this file? Should they be reverted?

Reply:
These are actually related to the test-pr, this diff should go away when this gets refreshed to the new master

Reply:
Actually I'm wrong, changing this back